html/cross-origin-embedder-policy/shared-workers.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
```

Comment:
Per https://github.com/whatwg/html/issues/4930 COEP will likely change to require a secure context, so this file should have an `.https` filename flag.

Reply:
That issue is not yet fixed. I've commented about this there: https://github.com/whatwg/html/issues/4930#issuecomment-762313885